Emby.Server.Implementations/SyncPlay/SyncPlayManager.cs

```diff
@@ -41,338 +44,412 @@ public class SyncPlayManager : ISyncPlayManager, IDisposable
         /// <summary>
         /// The map between sessions and groups.
         /// </summary>
-        private readonly Dictionary<string, ISyncPlayController> _sessionToGroupMap =
-            new Dictionary<string, ISyncPlayController>(StringComparer.OrdinalIgnoreCase);
+        private readonly Dictionary<string, IGroupController> _sessionToGroupMap =
+            new Dictionary<string, IGroupController>(StringComparer.OrdinalIgnoreCase);
 
         /// <summary>
         /// The groups.
         /// </summary>
-        private readonly Dictionary<Guid, ISyncPlayController> _groups =
-            new Dictionary<Guid, ISyncPlayController>();
+        private readonly Dictionary<Guid, IGroupController> _groups =
+            new Dictionary<Guid, IGroupController>();
 
         /// <summary>
-        /// Lock used for accesing any group.
+        /// Lock used for accesing the list of groups.
         /// </summary>
         private readonly object _groupsLock = new object();
 
+        /// <summary>
+        /// Lock used for accesing the session-to-group map.
```

Comment:
```suggestion
        /// Lock used for accessing the session-to-group map.
```

Reply:
Can I stuff these into the next PR or should I make a commit here now?

Reply:
commit here,